pypeit/core/wavecal/autoid.py

```diff
@@ -1617,25 +1764,21 @@ def __init__(self, spec, lamps, par=None, ok_mask=None, islinelist=False,
         self._debug = debug
         self._verbose = verbose
 
-        # Load the linelist to be used for pattern matching
+        # Line list provided? (not recommended)
```

Comment:
Should we add something to the docstring to say that this is not recommended, and the preferred option?

Reply:
warning added